coreprofile/_index.md

```diff
@@ -1,9 +1,8 @@
 ---
 title: "Jakarta EE Core Profile"
-weight: -3
-summary: "The Jakarta EE Core Profile defines a profile of the Jakarta EE platform specifically targeted at smaller runtimes.
-"
+weight: -5
+summary: "The Jakarta EE Core Profile defines a profile of the Jakarta EE Platform targetting cloud native microservice applications."
 #<!--.................0123456789.123456789.123456789.123456789.123456789.123456789-->
-summary_sixty_char: "Core profile of platform for smaller runtimes"
+summary_sixty_char: "Core profile of platform for Jakarta EE cloud native apps"
```

Comment:
Same as above. The "smaller runtimes" was sort of a compromise?

Reply:
Changed